src/frontend/src/flows/addDevice/manage/addRemoteDevice.ts

```diff
@@ -0,0 +1,55 @@
+import { AuthenticatedConnection } from "../../../utils/iiConnection";
+import { pollForTentativeDevice } from "./pollForTentativeDevice";
+import { displayError } from "../../../components/displayError";
+import { verifyTentativeDevice } from "./verifyTentativeDevice";
+import { withLoader } from "../../../components/loader";
+
+// The flow for adding a remote (i.e. other browser, non-hardware) device
+export const addRemoteDevice = async ({
+  userNumber,
+  connection,
+}: {
+  userNumber: bigint;
+  connection: AuthenticatedConnection;
+}): Promise<void> => {
+  // Enter registration mode and get info about the anchor
+  const [timestamp, anchorInfo] = await withLoader(() =>
```

Comment:
1. the function use to be `async`, not needed anymore?

```
await withLoader(withLoader(async () => 
```

2. nth: even if deferred I would not be against specifying the return type for readability

```
const [timestamp, anchorInfo] = await withLoader((): Promise<[something, something]> => ...
```

Reply:
> the function use to be async, not needed anymore?

Not needed; wasn't needed before either as far as I could tell

> nth: even if deferred I would not be against specifying the return type for readability

Sure, I'll actually add it to the LHS instead if ok for you (i.e.` const [ ... ] : [Timestamp, ...] =`)

Reply:
Done!